static/app/views/discover/eventDetails/content.tsx

```diff
@@ -147,13 +127,13 @@ class EventDetailsContent extends DeprecatedAsyncComponent<Props, State> {
           })
         : null;
 
-    const eventJsonUrl = `/api/0/projects/${organization.slug}/${this.projectId}/events/${event.eventID}/json/`;
+    const eventJsonUrl = `/api/0/projects/${organization.slug}/${projectId}/events/${event.eventID}/json/`;
 
     const hasProfilingFeature = organization.features.includes('profiling');
 
     const profileId = isTransaction(event) ? event.contexts?.profile?.profile_id : null;
 
-    const renderContent = (
+    const render = (
```

Comment:
Maybe more of a cleanup/nit thing but since this `render` no longer has logic, I feel like the contents should just be inlined.